_includes/releases/v22.2/v22.2.0-beta.5.md

```diff
@@ -0,0 +1,70 @@
+  - `jobs.row_level_ttl.range_total_duration` -> `jobs.row_level_ttl.span_total_duration`
+  - `jobs.row_level_ttl.num_active_ranges` -> `jobs.row_level_ttl.num_active_spans` [#90381][#90381]
+
+<h3 id="v22-2-0-beta-5-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug that resulted in additional empty bytes being written to files in S3 when the non-public `cloudstorage.s3.buffer_and_put_uploads.enabled` was set to true. [#89658][#89658]
+- Fixed a bug that caused internal errors in rare cases when running CTEs (statements with WITH clauses). This bug is only present in `v22.2.0-beta.2`, `v22.2.0-beta.3`, `v21.2.16`, and `v22.1.9`. [#89856][#89856]
+- CockroachDB will no longer silently truncate trailing characters when attempting to convert corrupt JSON string input into JSONB. [#89927][#89927]
+- The zone configs generated for a database with a secondary region were invalid. The `voter_constraints` and the lease preferences called for voters and leaseholders to exist in both the primary and secondary region, which is impossible. This bug has been present since `v22.2.0-alpha.1`. [#90186][#90186]
+- Changefeeds that use the setting `initial_scan_only` now ensure that all messages have successfully flushed to the sink prior to completion instead of potentially missing messages. [#90278][#90278]
+- Restoring a backup with a table containing `UniqueWithoutIndexConstraints` would fail because of incorrect `tableID` values being referenced in the constraints stored on the restored table. [#90052][#90052]
+- A bug has been fixed that caused incorrect results for queries with string similar filters (e.g., `col % 'abc'`) on tables with trigram indexes. This bug is only present in 22.2 pre-release versions up to and including `v22.2.0-beta.3`. [#90164][#90164]
+- A bug has been fixed that caused incorrect evaluation of comparison expressions involving time and interval types, like `col::TIME + '10 hrs'::INTERVAL' > '01:00'::TIME`. [#90367][#90367]
+- `REASSIGN OWNED BY` could run into errors, if any descriptor owned by the user is currently being dropped. [#90389][#90389]
+- Fixed a bug introduced in v22.1.9 that caused nodes to refuse to run jobs under rare circumstances. [#90464][#90464]
+- Fixed a bug (as described in [#90475](https://github.com/cockroachdb/cockroach/issues/90475) and [#90474](https://github.com/cockroachdb/cockroach/issues/90474)) where a restoring cluster's codec was accidentally used to reason about spans in the backup manifest during restore planning. [#90528][#90528]
+- Fixed a bug where some input to `crdb_internal.trim_tenant_prefix` would cause a node crash. [#90542][#90542]
+
+<h3 id="v22-2-0-beta-5-build-changes">Build changes</h3>
+
+- Added support for MacOS `arm64` (to support macOS systems with [Apple Silicon](https://support.apple.com/en-us/HT211814)). [#89944][#89944]
+- Cockroach Docker images are now multi-architecture manifests, supporting both the `x86_64` (`amd64`) and `arm64` architectures. [#90304][#90304]
```

Comment:
Duplicated from near the top of the file. I'd keep it here but with the edits I suggested above.